### DISCUSSION TITLE
Make Avatar Proxy Optional

### DISCUSSION THREAD

**Discussion by gaurishhs** [go-pkgz/auth#160] [Q&A] [answered]
Hi! is there a way to not use the avatar proxy and directly server the avatar urls obtained via providers

**Comment by umputun** [comment, marked as answer]
see https://github.com/go-pkgz/auth/blob/master/auth.go#L63

Alternatively you can set it to nil and it should have the same effect https://github.com/go-pkgz/auth/blob/master/auth.go#L132

**Reply by gaurishhs** [reply]
Alright. I just noticed that a user with same email cannot have multiple providers linked to it. Basically, If i signed in using google provider i'll have a different id and if i sign in using local provider ill have a different id irrespective of the fact that they both have the same emails. This is kinda really annoying..

**Reply by paskal** [reply]
True, that's the intention: you log in through some provider and you are unique you. If you're singing with another provider, it's someone else from the previously signed user. Lack of support for learning the account email is intentional.
